scripts/versions.ts

```diff
@@ -2,15 +2,17 @@ import path from 'node:path';
 import themes from '../themes.json';
 import { readFileSync, writeFileSync } from 'node:fs';
 
-const root = path.join(__dirname, '..');
+export const root = path.join(__dirname, '..');
 const versionsFileName = path.join(root, 'versions.json');
 
-function readJson(filename: string): { [key: string]: any } {
+export function readJson(filename: string): { [key: string]: any } {
   return JSON.parse(readFileSync(filename, { encoding: 'utf8' }));
 }
 
-function writeJson(filename: string, json: { [key: string]: any }) {
-  writeFileSync(filename, JSON.stringify(json, null, 2));
+export function writeJson(filename: string, json: { [key: string]: any }, addNewLine?: boolean) {
+  const stringifiedJson = JSON.stringify(json, null, 2);
+  const content = addNewLine ? `${stringifiedJson}\n` : stringifiedJson;
```

Comment:
Since the json files that get checked in are prettified by the pre-commit hook, is this logic necessary?

Reply:
The issue is that they're not getting prettified. It looks like the [lint-staged config](https://github.com/CMSgov/design-system/blob/main/package.json#L63-L75) doesn't pay attention to files that end with `.json`.

Reply:
Ah, darn

Reply:
yay! https://github.com/CMSgov/design-system/pull/3206

Reply:
Thanks for that super fast change @pwolfert - latest changes merge in main and undo the changes to writeJSON